<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagerDetails</class>
 <widget class="QWidget" name="PagerDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtGateway"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtNumber"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TextLabel11">
       <property name="text">
        <string>Provider:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="TextLabel12">
       <property name="text">
        <string>Email gateway:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbProvider"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="TextLabel10">
       <property name="text">
        <string>Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="Spacer7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtNumber</tabstop>
  <tabstop>cmbProvider</tabstop>
  <tabstop>edtGateway</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
